### PR TITLE
for the time being

### PR DESCRIPTION
**Pull request by Ashutosh741**

### CODE REVIEW

**Review by thakursachin05 — CHANGES_REQUESTED** [review]
code not working properly, show 404 page on random links 
show homepage when user go to /login
Always remove console.log before

File: src/pages/nopage/Nopage.jsx [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
-    </div>
-  )
-}
+    <section className="noPage">
+      <div id="notfound">
+        <div class="notfound-bg">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
```
Comment (thakursachin05) [inline src/pages/nopage/Nopage.jsx:13]: Why empty divs

File: src/pages/user/UserProfile.jsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ const UserProfile = () => {
       };
 
       const accessToken = JSON.parse(localStorage.getItem("accessToken")).token;
-      console.log("accesstoken", accessToken);
+      // console.log("accesstoken", accessToken);
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:59]: remove console

File: src/pages/user/UserProfile.jsx [hunk 3]
```diff
@@ -93,28 +91,43 @@ const UserProfile = () => {
...
-      // console.log("i'm soory bye bye")
       localStorage.removeItem("accessToken");
-      // Redirect to the login page
       Navigate("/login");
     }
   };
 
+  useEffect(()=>{
+    const storedUserData = JSON.parse(localStorage.getItem("user"));
+    // console.log("data browser",storedUserData)
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:120]: remove console

File: src/pages/nopage/Nopage.jsx (hunk 1 above)
Comment (Ashutosh741) [reply src/pages/nopage/Nopage.jsx:13]: css lga hai, un div mein

**Review by thakursachin05 — APPROVED** [review]
fix minor comments

**Pull request merged**